### PR TITLE
[DOCU-2779] Allow root path to be specified for Kong Manager

### PR DESCRIPTION
**Pull request by cloudjumpercat**
### Summary

Edited the `admin_gui_url` examples with the new root path for Kong Manager. Also added an entry to the changelog.
### Reason

DOCU-2779
### Testing

Go to:

- /gateway/latest/kong-manager/enable
- The release notes

### CODE REVIEW

**Comment by sumimakito** [comment]
@cloudjumpercat Hello. Thanks for your editing work!

Here are a few things I'd like to mention:
- We added a new config parameter `admin_gui_path` to kong.conf (a.k.a. `KONG_ADMIN_GUI_PATH` when configuring with env variables)
- When `admin_gui_path` presents, users are encouraged to configure `admin_gui_url` as well (however, we won't parse `admin_gui_path` from `admin_gui_url`)
- We have some constraints on `admin_gui_path`'s format:
	-  `admin_gui_path`  must start with a  `/`
	-   `admin_gui_path`  must not end with a  `/`  (except for the  `/`)
	-   `admin_gui_path`  can only contain letters, digits, hyphens (`-`), underscores (`_`), and slashes (`/`)
	-   `admin_gui_path`  must not contain continuous slashes (e.g.,  `//`  and  `///`)

**Comment by cloudjumpercat** [comment]
@sumimakito Thanks for all the info about the `admin_gui_path`! 

- I added that in to the examples, not sure if I formatted those correctly in the Docker and Linux instructions, if you wouldn't mind double checking those. 
- Also, even though you specify the path in `admin_gui_path`, do you still need to add the path to `admin_gui_url`? Or should `admin_gui_url` just be "http://localhost:8002"?

**Comment by sumimakito** [comment]
@cloudjumpercat Thanks again for your editing work! The updated documentation looks pretty good.

> I added that in to the examples, not sure if I formatted those correctly in the Docker and Linux instructions, if you wouldn't mind double checking those.

They look good to me.

> Also, even though you specify the path in `admin_gui_path`, do you still need to add the path to `admin_gui_url`? Or should `admin_gui_url` just be "http://localhost:8002"?

Yes. It's recommended to update the `admin_gui_url` when updating the `admin_gui_path`. This ensures links that appear in Kong Manager (e.g., DevPortal emails) are correct.

By the way, shall we use the phrase "base path" instead of "root path" to describe this feature? 

Thanks!

**Comment by sumimakito** [comment]
I also created a pull request to add the `admin_gui_path` parameter along with the description to Kong's default configuration.

**Review by Guaris — APPROVED** [review]

File: src/gateway/kong-manager/enable.md [hunk 1]
```diff
@@ -39,7 +41,7 @@ or hybrid mode), you can enable {{site.base_gateway}}'s graphical user interface
     kong restart -c {PATH_TO_KONG.CONF_FILE}
     ```
 
-3. Access Kong Manager on port `8002`.
+3. Access Kong Manager on port `8002` at the path you specified in `KONG_ADMIN_GUI_PATH`.
```
Comment (sumimakito) [inline src/gateway/kong-manager/enable.md:44]: nitpick: Would it be better if we use `admin_gui_path` here to keep the parameter name consistent with the ones in previous steps?

```suggestion
3. Access Kong Manager on port `8002` at the path you specified in `admin_gui_path`.
```

**Review by sumimakito — APPROVED** [review]
LGTM and thanks!

**Pull request merged**